src/app/GUI.java

```diff
@@ -336,6 +346,39 @@ private void setupGUI() {
     lbl_moveSelectXP = new JLabel("XP");
     lbl_moveSelectXP.setBounds(339, 50, 97, 17);
     panel_moveSelect.add(lbl_moveSelectXP);
+    
+    panel_moveInventory = new JPanel();
+    frame.getContentPane().add(panel_moveInventory, "name_29070680417700");
+    panel_moveInventory.setLayout(null);
+    
+    scrollPane_moveInventoryTable = new JScrollPane();
+    scrollPane_moveInventoryTable.setBounds(50, 61, 348, 91);
+    panel_moveInventory.add(scrollPane_moveInventoryTable);
+    
+    table_moveInventory = new JTable();
+    table_moveInventory.setModel(new DefaultTableModel(
+    	new Object[][] {
+    		{null, null, null},
+    		{null, null, null},
+    		{null, null, null},
+    		{null, null, null},
+    	},
+    	new String[] {
+    		"Name", "Level", "XP"
+    	}
+    ));
+    table_moveInventory.setFont(new Font("Times New Roman", Font.PLAIN, 12));
```

Comment:
Do you want to keep it as Times New Roman? The other ones are in Dialog - I don't really mind, as long as it's consistent.